sources/packages/web/src/views/student/StudentOverawardsBalance.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <student-page-container layout-template="centered" hide-restriction="true">
+    <template #header>
+      <header-navigator
+        title="Home"
+        subTitle="Overawards Balance"
+        :routeLocation="{ name: StudentRoutesConst.STUDENT_DASHBOARD }"
+      />
+    </template>
+    <body-header-container :enableCardView="true">
```

Comment:
@andrewsignori-aot @ann-aot @andrepestana-aot @sh16011993 

I believe this is a peculiar scenario (cards with image) where we cannot use the `body-header-container` 
due to the need to alignment of the image besides the header instead of underneath the header. 

![image](https://user-images.githubusercontent.com/54600590/228388269-c4ffd6a0-a7fc-416a-9cf5-062652f27cc4.png)

Should we plan to update the body-header-container to accommodate this content?

Reply:
@dheepak-aot I would prefer just to ask @hellolynn-tbtb if the image cannot be aligned a bit to the bottom.

Reply:
talked to @hellolynn-tbtb  and moved the image down in the figma and used the body-header.

Reply:
Thanks @guru-aot! Feel free to shoot me a quick message next time :) @dheepak-aot